src/components/course/routes/ExternalCourseEnrollment.jsx

```diff
@@ -46,15 +70,34 @@ const ExternalCourseEnrollment = () => {
               <h2 className="mb-3">
                 Your registration(s)
               </h2>
-              <p className="small bg-light-500 p-3 rounded-lg">
-                <strong>
-                  This is where you finalize your registration for an edX executive
-                  education course through GetSmarter.
-                </strong>
-                &nbsp; Please ensure that the course details below are correct and confirm using Learner
-                Credit with a &quot;Confirm registration&quot; button.
-                Your Learner Credit funds will be redeemed at this point.
-              </p>
+              {isDuplicateOrder(formSubmissionError) && (
+                <Alert
+                  variant="success"
```

Comment:
[suggestion] We may want to always render this `Alert`, but conditionally display it through its `show` prop, i.e. `show={isDuplicateOrder(formSubmissionError)}`.